sys-cluster/glusterfs/files/glusterd-10.2.initd

```diff
@@ -0,0 +1,32 @@
+#!/sbin/openrc-run
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+description="Gluster elastic volume management daemon"
+command="/usr/sbin/glusterd"
+pidfile="/run/${SVCNAME}.pid"
+command_args="-N"
+
+command_background="yes"
+
+depend() {
+	need net
+	before netmount
+}
+
+start_pre() {
+	# Ensure that the GlusterFS auxiliary mount parent directory exists
+	checkpath --directory --owner gluster:gluster --mode 0775 /run/gluster
+}
+
+start_post() {
+	local c=0
+	ebegin "Waiting for glusterd to start up"
+	while ! /usr/sbin/gluster volume list >/dev/null 2>&1 && [ "${c}" -lt "${glusterd_max_wait_start-60}" ]; do
+		$(( ++c ))
```

Comment:
```
 * QA Notice: shell script appears to use non-POSIX feature(s):
 *    possible bashism in /etc/init.d/glusterd line 26 ('$((++n))' should be '$((n=n+1))'):
 *    		$(( ++c ))
```
You can install dev-util/checkbashisms to run these checks.

Reply:
Thank you!!!!!!